playwright-e2e/pages/page-base.ts

```diff
@@ -73,8 +73,20 @@ export default abstract class PageBase implements PageInterface {
     return this.page.goto(`${this.baseUrl}${urlPath}`, { waitUntil: 'load' });
   }
 
-  protected async clickAndWaitForNav(locator: Locator, opts: { waitForNav?: boolean } = {}): Promise<void> {
-    const { waitForNav = false } = opts;
+  /**
+   *
+   * @param {Locator} locator
+   * @param {{waitForNav?: boolean, forceClick?: boolean}} opts
+   *  <br> - forceClick: Whether to bypass error message checks before click. If set to true, click regardless an error on page. Defaults to false.
+   *  <br> - waitForNav: Whether to wait for page navigation to complete after click. Defaults to false.
+   * @returns {Promise<void>}
+   * @protected
+   */
+  protected async clickAndWaitForNav(locator: Locator, opts: { waitForNav?: boolean; forceClick?: boolean } = {}): Promise<void> {
+    const { waitForNav = false, forceClick = false } = opts;
+    if (!forceClick) {
+      await expect(this.page.locator('.error-message')).toBeHidden();
+    }
```

Comment:
Add new parameter `forceClick` to address one flaky failure I saw during testing. An error message did not go away for 10 seconds, so the click a button has failed. I don't know why it happened.